packages/node/src/parsers.ts

```diff
@@ -99,8 +120,12 @@ export async function extractStackFromError(error: Error): Promise<stacktrace.St
 }
 
 /** JSDoc */
-export async function parseStack(stack: stacktrace.StackFrame[]): Promise<StackFrame[]> {
+export async function parseStack(stack: stacktrace.StackFrame[], options?: NodeOptions): Promise<StackFrame[]> {
   const filesToRead: string[] = [];
+
+  const linesOfContext =
+    options && options.frameContextLines !== undefined ? options.frameContextLines : DEFAULT_LINES_OF_CONTEXT;
```

Comment:
We should `parseInt` here, just to be safe

Reply:
Typescript complains it's already a number, let's leave it like this we do not validate any other options.